main.py

```diff
@@ -43,12 +43,14 @@ def main():
 							print('3')
 						elif account_choice == 4:
 							email = input("Enter email to remove: ")
-							
+							print(queries.add_new_email(email))
```

Comment:
This should call queries.remove_email() not add_new_email(). Otherwise get a runtime error.